TheWarWithin/MageFire.lua

```diff
@@ -1739,6 +1793,14 @@ spec:RegisterAbilities( {
                     if buff.expanded_potential.up then removeBuff( "expanded_potential" )
                     else
                         removeBuff( "hot_streak" )
+                        if talent.spellfire_spheres.enabled then
+                            if buff.next_blast_spheres.stacks == 5 then
```

Comment:
Same questions; does it trigger at 5 stacks?  If so, the check should be 4 stacks since this cast gives you the 5th stack.
And does it need to vary for Rondumancy or whatever?